pkgs/tools/networking/xdp-tools/default.nix

```diff
@@ -74,7 +66,7 @@ stdenv.mkDerivation rec {
     homepage = "https://github.com/xdp-project/xdp-tools";
     description = "Library and utilities for use with XDP";
     license = with licenses; [ gpl2 lgpl21 bsd2 ];
-    maintainers = with maintainers; [ tirex vcunat ];
+    maintainers = with maintainers; [ tirex vcunat vifino];
```

Comment:
no space at the end looks a bit ugly

Reply:
Ah, yeah, of course. I'll fix it. alejandra would remove the spaces on both sides.